lib/resty/expr/v1.lua

```diff
@@ -136,6 +143,7 @@ local compare_funcs = {
     end,
     ["in"] = in_array,
     ["has"] = has_element,
+    ["ipmatcher"] = ip_matcher,
```

Comment:
It would be better to avoid using nouns as an operator. `Client address ipmatcher 127.0.0.1` is not a good sentence.

Reply:
what about `match`?

Reply:
> what about `match`?

`ipmatch` would be better?

Reply:
SGTM